### PR TITLE
Updated grafana secret name

### PR DESCRIPTION
**Pull request by mukrishn**
### Description
There is a change in the secret name on 4.10 releases, so included a line to fetch the secret automatically.

### CODE REVIEW

**Comment by mukrishn** [comment]
@rsevilla87 PTAL, thanks!

**Comment by rsevilla87** [comment]
> @rsevilla87 PTAL, thanks!

Hey @mukrishn, have you verified it works in older versions?

**Comment by mukrishn** [comment]
> Hey @mukrishn, have you verified it works in older versions?

Yes @rsevilla87, tested it on 4.9 and 4.10.

**Review by rsevilla87 — APPROVED** [review]
LGTM!

File: dittybopper/deploy.sh [hunk 1]
```diff
@@ -108,8 +108,9 @@ fi
 #FIXME: This is OCP-Specific; needs updating to support k8s
 echo ""
 echo -e "\033[32mGetting environment vars...\033[0m"
-export PROMETHEUS_URL=$($k8s_cmd get secrets -n openshift-monitoring grafana-datasources -o go-template='{{index .data "prometheus.yaml" | base64decode }}' | jq '.datasources[0].url')
-export PROMETHEUS_PASSWORD=$($k8s_cmd get secrets -n openshift-monitoring grafana-datasources -o go-template='{{index .data "prometheus.yaml"| base64decode }}' | jq 'if .datasources[0].basicAuthPassword != null then .datasources[0].basicAuthPassword else .datasources[0].secureJsonData.basicAuthPassword end')
+export GRAFANA_SECRET=$($k8s_cmd get secrets -n openshift-monitoring -l app.kubernetes.io/component=grafana | grep datasource | awk '{print $1}')
```
Comment (rsevilla87) [inline dittybopper/deploy.sh:111]: Small nit, we don't need to export GRAFANA_SECRET

Comment (mukrishn) [reply dittybopper/deploy.sh:111]: you are right, I will correct that.

**Pull request merged**